src/plugins/eui_utils/README.md

```diff
@@ -0,0 +1,65 @@
+# EuiUtils
+
+The EuiUtils plugin is a way to create easier integration of EUI colors, themes, and other utilities with Kibana. They usually take into account the current theme (light or dark) of Kibana and return the correct object that was asked for.
+
+## EUI plus Elastic-Charts
+
+EUI provides a light and dark theme object to work with Elastic-Charts. However, every instance of a Chart would need to pass down this the correctly EUI theme depending on Kibana's light or dark mode. There are several ways you can use EuiUtils to grab the correct theme.
+
+### `useChartsTheme`
+
+The simple fetching of the correct EUI theme; a **React hook**.
+
+```js
+import { npStart } from 'ui/new_platform';
+import { Chart, Settings } from '@elastic/charts';
+
+<Chart>
+  <Settings theme={npStart.plugins.eui_utils.useChartsTheme()} />
+</Chart>;
```

Comment:
Might want to put this example in a function component:

```ts
export const MyComponent = () => (
  <Chart>
    <Settings theme={npStart.plugins.eui_utils.useChartsTheme()} />
  </Chart>
);
```

Reply:
Done. Thanks! Will update on passing test